lib/src/web_ui/code_input.dart

```diff
@@ -145,8 +202,75 @@ class CodeInput {
     return strIndex + 2;
   }
 
+  ///Find the list of words that contain the same prefix as currWord
+  List<String> _wordMatches(String currWord, bool fullWord) {
+    List<String> matchingWords = [];
+    int currLength = currWord.length;
+    for (String schemeWord in wordToDocs.keys) {
+      if (((schemeWord.length > currWord.length) && !fullWord) ||
+          schemeWord.length == currWord.length) {
+        if (schemeWord.substring(0, currLength) == currWord) {
```

Comment:
The declared name in line 208 isn't really much shorter and doesn't save any repeated work (plus we already use `currWord.length` in the two lines above), so just replace `currLength` with `currWord.length` here and eliminate the declaration.